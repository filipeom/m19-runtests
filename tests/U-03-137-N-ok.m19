#f!(<%>p) = 0 { p[0]! }
#m19!() = 0 {
  <%>p = [3];
  p[0] = 30;
  p[1] = 61;
  p[2] = 71;
  apply f -> p : 1 .. 2;
}

