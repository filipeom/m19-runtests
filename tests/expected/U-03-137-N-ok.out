6.1E17.1E1
